torchsynth/config.py

```diff
@@ -0,0 +1,37 @@
+import torch
+
+
+def check_for_determinism():
+    """
+    This checks to make sure that the expected random results
+    are produced by torch.rand when seeded. This raises a warning
+    indicating if reproducibility is not guaranteed.

```

Comment:
Add a comment with the following details:
1) CPU vs GPU torch.rand gives different results (https://discuss.pytorch.org/t/deterministic-prng-across-cpu-cuda/116275)
2) Profiling indicated we could do torch.rand on CPU, faster than torchcsprng